src/Infrastructure.Windows/SoftwareManagers/HeroicManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace AutoGame.Infrastructure.Windows.SoftwareManagers;
+
+using System;
+using System.IO;
+using System.IO.Abstractions;
+using AutoGame.Core.Interfaces;
+using AutoGame.Infrastructure.Windows.Interfaces;
+using AutoGame.Infrastructure.Windows.Services;
+using Serilog;
+
+internal sealed class HeroicManager : ISoftwareManager
+{
+    public HeroicManager(
+        IUser32Service user32Service,
+        IProcessService processService
+       )
+    {
+        this.User32Service = user32Service;
+        this.ProcessService = processService;
+
+    }
+
+
+    private IUser32Service User32Service { get; }
+    private IProcessService ProcessService { get; }
+
+
+    public string Key => "Heroic";
+
+    public string Description => "Heroic Fullscreen";
```

Comment:
You might change this to "Heroic Games Launcher" since that's the official title.